hasher-matcher-actioner/hmalib/common.py

```diff
@@ -12,3 +23,132 @@ def get_logger(name=__name__, level=logging.INFO):
     logger = logging.getLogger(name)
     logger.setLevel(level)
     return logger
+
+
+logger = get_logger(__name__)
+s3_client = boto3.client("s3")
+
+THREAT_EXCHANGE_DATA_BUCKET_NAME = os.environ["THREAT_EXCHANGE_DATA_BUCKET_NAME"]
+THREAT_EXCHANGE_DATA_FOLDER = os.environ["THREAT_EXCHANGE_DATA_FOLDER"]
+THREAT_EXCHANGE_PDQ_FILE_EXTENSION = os.environ["THREAT_EXCHANGE_PDQ_FILE_EXTENSION"]
```

Comment:
Nit: Put all the module level attributes at the to level (except logger).

Maybe blocking: Overall, this all feels like it belongs in another module called s3. Maybe common should be a folder and not a file? common/logging.py ?

Nit: I'm partial to using a wrapping class for this (https://github.com/Dcallies/ThreatExchange/blob/a5df357bd9d86243b562984d9030915a30f2980b/hasher-matcher-actioner/hmalib/lambdas/fetcher.py#L13), because it makes it easier to override the functionality for testing - relying entirely on environment variables can be very error prone.

Reply:
I like the class idea. Will add. 

Also agree about moving but think that should happen in another PR to keep comments